jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ExtraDirectoryParameters.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.gradle;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import javax.inject.Inject;
+import org.gradle.api.Project;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.Internal;
+
+/** Configuration of an extra directory. */
+public class ExtraDirectoryParameters {
+
+  private Project project;
+  private Path from;
+  private String into;
+
+  @Inject
+  public ExtraDirectoryParameters(Project project) {
+    this.project = project;
+    from = Paths.get("");
+    into = "/";
```

Comment:
Instead of initializing here, how about initializing them in the field definitions? Other ...Parameters follow this convention. This also ensures they are always initialized even when using a potentially different constructor.